lib/nsfunc/application-navigation-api/resolve-url.js

```diff
@@ -20,9 +20,12 @@ exports.nlapiResolveURL = (type, identifier, id, displayMode) => {
 
     let db = global.$db;
     if (~['suitelet', 'restlet'].indexOf(type)) {
-        if (!identifier) throw nlapiCreateError('SSS_IDENTFIER_ARG_REQ');
+        if (!identifier) throw nlapiCreateError('SSS_MISSING_REQD_ARGUMENT');
+        if (!id) throw nlapiCreateError('INVALID_ID');
+        if (!isNaN(Number(id))) throw Error('Number argument on nlapiResolveURL 3rd parameter with 1st' +
```

Comment:
this is not a netsuite error. I regards a specific uncommon scenario handled by netsuite but not by nsmockup.
In this case, we aknowledge that we do not handle this case (instead of the throwing an unhandled non-user friendly error elsewhere).